package app.gate;
import "hero.proto";

//通用请求，无参数
message CommonResponse
{
    required bool result = 1;
    optional string message = 2;
}

//请求武将列表
message HeroListResponse
{
    required bool result = 1;
    repeated Hero_PB hero_list = 2;
}



//武将升级
message HeroUpgradeResponse
{
    required int64 id = 1;
    required int32 level = 2;
    required int32 percent = 3;
}

